release-pr/src/neon_release_pr/cli.py

```diff
@@ -20,60 +20,56 @@ def main(
 ):
     """neon-release is a CLI tool for releasing components at neon"""
     ctx.dry_run = dry_run
+    git.ready()
 
 
 @app.command()
 def new(
-    component: Annotated[str, typer.Argument(envvar="NEON_RELEASE_PR_COMPONENT")],
+    component: Annotated[str, typer.Argument(metavar="COMPONENT", show_default=False)],
+    cherry_pick: Annotated[
+        Optional[list[str]],
+        typer.Argument(
+            metavar="COMMIT...",
+            show_default=False,
+            help="Commit(s) to cherry-pick onto the release branch. Implies --base set to the previous release.",
+        ),
+    ] = None,
     base: Annotated[
         Optional[str],
-        typer.Option(metavar="REF", help="Base to release from.", show_default="main"),
-    ] = None,
-    hotfix: Annotated[
-        bool,
-        typer.Option(
-            "--hotfix",
-            help="Shortcut: set base to <release-branch>.",
-            envvar="NEON_RELEASE_PR_HOTFIX",
-        ),
-    ] = False,
-    cherry_pick: Annotated[
-        list[str],
         typer.Option(
-            "--cherry-pick",
-            metavar="COMMIT",
-            help="Cherry-pick commits (can be used multiple times).",
-            envvar="NEON_RELEASE_PR_CHERRY_PICK",
+            "--base",
+            "--from-commit-sha",
+            metavar="REF",
+            help="Base to release from.",
+            show_default="main",
         ),
-    ] = [],
+    ] = None,
     auto_merge: Annotated[
         bool,
         typer.Option(
             "--auto-merge",
+            "--automatic",
             help="Enable auto-merge after PR approval.",
-            envvar="NEON_RELEASE_PR_AUTO_MERGE",
         ),
     ] = False,
     auto_approve: Annotated[
         bool,
         typer.Option(
             "--auto-approve",
```

Comment:
An optional one:

```suggestion
            "--approve",
```
I know it's not related to the changes, but should it be just `--approve`?

Reply:
Sure, why not.